distro/README.md

```diff
@@ -0,0 +1,70 @@
+# riff-distro
+
+The `riff-distro` CLI provides commands to create a distribution of riff.
+It uses a separate binary so that regular riff CLI users are not exposed to commands they'll never use, but it shares code and manifests (like `stable`) with the riff CLI. 
+
+### To build riff-distro from source
+
+Clone the riff project, and ensure that you have a working go environment with `dep`.
+
+```sh
+cd distro
+make build   # build riff-distro
+make install # copy to $GOPATH/bin
+```
+
+## To create a new distribution of riff
+
+Start in an empty directory
+```sh
+mkdir ~/release-new
+cd ~/release-new
+```
+
+Download a stable release manifest and the corresponding release yaml files.
+```sh
+riff-distro system download -m stable -o .
+```
+
+Write a list of all images found in the release yaml files into `image-manifest.yaml`.  
+Use ` --no-check` to avoid checking each image by pulling it down with your local docker daemon.
+```sh
+riff-distro image list -m stable
+```
```

Comment:
We should probably include a warning here that the list of images needs to be validated. See the help from `riff-distro image list`.